packages/contracts-ops/contracts/scripts/RotateUpdater.s.sol

```diff
@@ -52,38 +52,35 @@ contract RotateUpdaterLogic is Config, CallBatch {
     }
 }
 
-contract RotateUpdater is Script, RotateUpdaterLogic {
+abstract contract RotateUpdater is Script, RotateUpdaterLogic {
```

Comment:
why is this now abstract?

Reply:
to silence compiler warnings - as i said in slack, i thought it still allows you to run the script?